src/docs/cookbook/maintenance/error-reporting.md

```diff
@@ -47,138 +45,64 @@ To get a DSN, use the following steps:
 
   1. [Create an account with Sentry][].
   2. Log in to the account.
-  3. Create a new app.
-  4. Copy the DSN.
+  3. Create a new Flutter project.
+  4. Copy the code snippet which includes the DSN.
 
 ## 2. Import the Sentry package
 
-Import the [`sentry`][] package into the app.
+Import the [`sentry_flutter`][] package into the app.
```

Comment:
[The `sentry` package is for pure Dart apps and the new `sentry_flutter`](https://pub.dev/packages?q=sentry) takes dependency on Flutter and integrates with it, adding native support:

![image](https://user-images.githubusercontent.com/1633368/107383851-66bea500-6abf-11eb-9111-4cf152df9e7a.png)